lib/assets/javascripts/twine.coffee

```diff
@@ -56,6 +56,7 @@
 
   bind = (context, node, indexes, forceSaveContext) ->
     currentBindingCallbacks = []
+    element = null
```

Comment:
Is this actually needed? The js should have a `var element` at the top even if we don't initialize it to null